packages/host/src/AppDrawer.tsx

```diff
@@ -78,7 +79,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     [theme.breakpoints.up('sm')]: {
       width: theme.spacing(9),
     },
-    '& li > a': { borderRadius: 20, height: 40, padding: 10, width: 40 },
+    '& li > a': { borderRadius: 20, height: 32, padding: 10, width: 40 },
```

Comment:
but why? now the circles aren't circles - in Manu's design the closed drawer had these as 40px circles and the larger version as 32px high lozenges

Reply:
Ah right, sorry. Reverted: b3995da

I was trying to adjust this thing I noticed which irrationally annoyed me.. the shifting of the menu items due to the height change

![height-change](https://user-images.githubusercontent.com/35858975/128095937-ba8e77f8-a2df-4a4b-bbe7-29c3fab809e7.gif)

V

![height-change-no](https://user-images.githubusercontent.com/35858975/128096110-193a2d68-45ab-40dd-98f9-38ae64c9b625.gif)

Reply:
yeah, bothered me too.
it isn't perfect - but this is pretty good:
```
    '& li > a': {
      borderRadius: 20,
      height: 40,
      margin: '12.5px 0',
      padding: 10,
      width: 40,
    },
```
under `drawerPaperClose`. Makes the circles bigger and round, while keeping their vertical position